pkg/util/ingress/ingress_test.go

```diff
@@ -261,18 +262,18 @@ func TestValidateIngressPath(t *testing.T) {
 			copyIng:         generateDumbIngressforPathTest(&pathTypeImplSpecific, "/foo/bar/(.+)"),
 		},
 		{
-			name:                      "should accept path with regex characters and pathType exact, but pathType validation disabled",
-			wantErr:                   false,
-			additionalChars:           defaultAdditionalChars,
-			disablePathTypeValidation: true,
-			copyIng:                   generateDumbIngressforPathTest(&pathTypeImplSpecific, "/foo/bar/(.+)"),
+			name:                     "should accept path with regex characters and pathType exact, but pathType validation disabled",
+			wantErr:                  false,
+			additionalChars:          defaultAdditionalChars,
+			EnablePathTypeValidation: true,
```

Comment:
This one should error, if it is not something is wrong.

Validation is enabled, so you shouldn't be supposed to be passing regex chars on it as you defined it is pathtype exact